### ISSUE TITLE
Support contenteditable="plaintext-only"

### ISSUE THREAD

**Issue by JohnLouderback** [hl037/vue-contenteditable#3]
Besides true, in WebKit descendant browsers, contenteditable can be equal to the string "plaintext-only". It would be good if this was support. It works currently, except the warning logged by Vue because it isn't a boolean type.

**Comment by hl037** [comment]
Thanks for the proposal,
it seems not to be supported on FF, so I'm not sure if it's good idea. I'll also have to check how it would behave with `noHTML` and `noNL`

**Comment by JohnLouderback** [comment]
Yeah, I believe it does not work in FF. My particular use case is within Electron, so it's fine for me to use. From what I can see, everything works as expected. The only issue I'm running into is that Vue logs a warning that I pass a string versus a boolean, which creates visual noise in my logs even though in my environment, a string is valid, albeit non-standard.
